File: src/main/java/com/fretron/terminalModels/OtherTrainBooking.java
package com.fretron.terminalModels;

import com.fretron.Model.StandardMeasurement;
import com.google.gson.GsonBuilder;

public class OtherTrainBooking {

    private String operatorName;
    private StandardMeasurement quantity;

    public OtherTrainBooking() {
    }

    public OtherTrainBooking(String operatorName, StandardMeasurement quantity) {
        this.operatorName = operatorName;
        this.quantity = quantity;
    }

    public void setOperatorName(String operatorName) {
        this.operatorName = operatorName;
    }

    public String getOperatorName() {
        return operatorName;
    }

    public void setQuantity(StandardMeasurement quantity) {
        this.quantity = quantity;
    }

    public StandardMeasurement getQuantity() {
        return quantity;
    }

    @Override
    public String toString() {
        return new GsonBuilder().serializeNulls().create().toJson(this);
    }
}

File: src/main/java/com/fretron/terminalModels/OtherTrainBooking.kt
package com.fretron.terminalModels

import com.fretron.Model.StandardMeasurement
import com.google.gson.GsonBuilder

data class OtherTrainBooking(
    var operatorName: String?,
    var quantity: StandardMeasurement?,
    var tripId : String? ,
    var terminalId : String?,
    var uuid: String?
) {

    constructor() : this(operatorName = null, quantity =  null , tripId = null , terminalId = null,
    uuid = null)

    override fun toString(): String {
        return GsonBuilder().serializeNulls().create().toJson(this)
    }
}